expand environment variables in paths defined in the config.ini file
Author: Samuel Moors, Vrije Universiteit Brussel (VUB)
diff -ur MuPeXI-1.2.0.orig/MuPeXI.py MuPeXI-1.2.0/MuPeXI.py
--- MuPeXI-1.2.0.orig/MuPeXI.py	2018-04-17 14:20:46.000000000 +0200
+++ MuPeXI-1.2.0/MuPeXI.py	2018-10-13 21:34:08.883677229 +0200
@@ -165,7 +165,7 @@
     # parse config file
     config = SafeConfigParser()
     config.read(config_file)
-    path = config.get(category, ID) if config.has_option(category, ID) else None
+    path = os.path.expandvars(config.get(category, ID)) if config.has_option(category, ID) else None
 
     return path
 
